src/components/camera-tool.js

```diff
@@ -620,12 +621,15 @@ AFRAME.registerComponent("camera-tool", {
           // HACK, bone visibility typically takes a tick to update, but since we want to be able
           // to have enable() and disable() be reflected this frame, we need to do it immediately.
           boneVisibilitySystem.tick();
+          boneVisibilitySystem.updateMatrices();
         }
 
         const tmpVRFlag = renderer.xr.enabled;
         const tmpOnAfterRender = sceneEl.object3D.onAfterRender;
+        const tmpAutoUpdate = sceneEl.object3D.autoUpdate;
         delete sceneEl.object3D.onAfterRender;
         renderer.xr.enabled = false;
+        sceneEl.object3D.autoUpdate = false;
```

Comment:
Might be good to add a comment here explaining why we are turning off autoUpdate.

Reply:
Added, thanks